### PR TITLE
My first pair programming

### PR DESCRIPTION
**Pull request by Grandi0z**
###In this code we test our HTML general  knowledge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 id="Header" class="flex-container">
+      <img id ="header-img" src="https://i.annihil.us/u/prod/marvel/i/mg/c/80/4d5c2e7075ded/clean.jpg" alt="logo" width="100" height="70">logo
+      <nav id="nav-bar">
+            <a class="nav_link" href="#">cat</a>
+            <a class="nav_link" href="#">Dog</a>
+            <a class="nav_link" href="#">Pig</a>
```
Comment (Grandi0z) [inline index.html:29]: Please put <a> tags inside in an <ul> as <li>.  I think it looks more semantic.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    -->
+    <h1 id="Header" class="flex-container">
+      <img id ="header-img" src="https://i.annihil.us/u/prod/marvel/i/mg/c/80/4d5c2e7075ded/clean.jpg" alt="logo" width="100" height="70">logo
+      <nav id="nav-bar">
+            <a class="nav_link" href="#">cat</a>
+            <a class="nav_link" href="#">Dog</a>
+            <a class="nav_link" href="#">Pig</a>  
+      </nav>
+    </h1>
```
Comment (Grandi0z) [inline index.html:31]: You have a beautiful design. However, you will agree with me that using <header> tag to wrap h1 and nav tags is semantically more beautiful.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<video width="320" height="240" controls>
+  <source src="https://youtu.be/Yd47Z8HYf0Y" type="video/mp4">
```
Comment (Grandi0z) [inline index.html:33]: It seems that you had a great idea to add a video file to your website 👍 . Please read this article which explains step by step how to do it perfectly 😄   :  https://www.w3schools.com/html/html_youtube.asp

**Review by Grandi0z — COMMENTED** [review]
# Status: Changes Required ♻️ 
Please read my comment and make the required changes.

ask for a re-review when done.

Cheese, thank you 😄 
6lQ3

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Grandi0z   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**